Updated available services.

patrickramp opened the pull request:
# PR: Thank you!

## Reason

- [x] Fix: Wrong information
- [ ] Fix: Typo error
- [ ] Fix: License violation
- [x] Add: New Item

## Type

- [ ] Node.js Hosting
- [x] VPS
- [ ] Static Hosting
- [x] DB Hosting

## Name


Removed F(x)Data Cloud (appears to be defunct and link not working)
Added [HETZNER](https://www.hetzner.com/) VPS
Added [CockRoachDB](https://www.cockroachlabs.com/) Database service 


## Thank you for improving the project

**This helps others find VPS or hosting of their choice**

dalisoft commented:
@patrickramp Thank you for PR

1. HETZNER is duplicate as it's already mentioned
2. Database section **Open source** means there any bonus/free credit for **Open source** developers

Please fix these case and then i can merge PR

patrickramp commented:
@dalisoft Sorry about that,
Updated Hetzner pricing, links, and ranking. Removed redundant Hetzner entry.
Removed Open source comments on CockroachDB.

dalisoft commented:
@patrickramp Please merge master and resolve conflicts. 
And format markdown as VSCode + Prettier default modes without custom configs on your IDE/Editor
